### DISCUSSION TITLE
Umstieg Debmatic Raspberrymatic Backup einspielen

### DISCUSSION THREAD

**Discussion by h2okopfmt** [OpenCCU/OpenCCU#1947] [🇩🇪 Fragen&Antworten]
### Describe the issue you are experiencing

Ich möchte von meiner VM unter Proxmos Debmatic Version : 3.65.8.95
Der Anschluss der HM Platine ist Via USB.

Auf einen Docker Raspberrymatic Version : 3.65.8.20220831
Der Anschluss der HM Platine ist Via USB von der VM umgesteckt

Umsteigen.

Leider bekomme ich das Backup von der Debmatic nicht in die Rasberrymatic eingelesen bzw wenn Geräte erkannt werden nur die IP Geräte.

Weiss jemand einen Rat warum die HM Module nicht erkannt werden ? 
Was muss gemacht werden damit die HM Modul erkannt werden ? 
Leider finde ich nichts im weiten WWW oder bin zu blöd zum suchen , kann ja auch sein.

VG Markus 

### Describe the behavior you expected

Geräte fehlen ,

### Steps to reproduce the issue

1. In debmatic Backup erstellen
2. In Rasberrymatic Backup einspielen
3. Keine HM Geräte in der Raspberrymatic
...

### What is the version this bug report is based on?

3.65.8.20220831

### Which base platform are you running?

rpi4 (RaspberryPi4)

### Which HomeMatic/homematicIP radio module are you using?

HmIP-RFUSB

### Anything in the logs that might be useful for us?

```
Identifying host system: oci, OK
Initializing RTC Clock: no hardware found
Running sysctl: OK
Checking for Factory Reset: not required
Checking for Backup Restore: not required
Initializing System: OK
Setup ca-certificates: OK
Starting logging: OK
Populating /dev using udev: done
Init onboard LEDs: init, OK
Starting irqbalance: OK
Starting iptables: OK
Starting network: eth0: link up, fixed, firewall, inet down, 192.168.55.223, OK
Identifying Homematic RF-Hardware: ...Starting watchdog...
Identifying host system: oci, OK
Initializing RTC Clock: no hardware found
Running sysctl: OK
Checking for Factory Reset: not required
Checking for Backup Restore: not required
Initializing System: OK
Setup ca-certificates: OK
Starting logging: OK
Populating /dev using udev: done
Init onboard LEDs: init, OK
Starting irqbalance: OK
Starting iptables: OK
Starting network: eth0: link up, fixed, firewall, inet down, 192.168.55.223, OK
Identifying Homematic RF-Hardware: .......HmRF: RPI-RF-MOD/HB-RF-USB@usb-0000:01:00.0-1.3, HmIP: RPI-RF-MOD/HB-RF-USB@usb-0000:01:00.0-1.3, OK
Updating Homematic RF-Hardware: RPI-RF-MOD: 4.4.22, not necessary, OK
Starting hs485dLoader: disabled
Starting xinetd: OK
Starting eq3configd: OK
Starting lighttpd: OK
Starting ser2net: disabled
Starting ssdpd: OK
Starting sshd: OK
Starting NUT services: disabled
Initializing Third-Party Addons: OK
Starting LGWFirmwareUpdate: ...OK
Setting LAN Gateway keys: /etc/config/CCU2GW0001.keychange OK
Starting SNMP daemon: OK
Starting hs485d: disabled
Starting multimacd: .OK
Starting rfd: ..OK
Starting HMIPServer: .........OK
Starting ReGaHss: .OK
Starting CloudMatic: OK
Starting NeoServer: disabled
Starting Third-Party Addons: OK
find: /usr/local/etc/monit*.cfg: No such file or directory
Starting crond: OK
Setup onboard LEDs: booted, OK
Finished Boot: 3.65.8.20220831 (raspmatic_oci_arm)
```

### Additional information

Naja Geräte fehlen

**Comment by ecdlguy** [comment]
Hallo, das würde mich auch interessieren.
Ich spiele mit dem Gedanken, von debmatic auf Raspberrymatic zu wechseln, da ich aufgrund des HA Addons keine weitere VM für debmatic mehr benötigen würden.
Daher die Frage: Sind Backups prinzipiell nicht zwischen debmatic und RBmatic kompatibel? Oder sollte sie es sein und hier ist im speziellen etwas schief gegangen?

Danke und Grüße, Thorsten

**Comment by sebastianohl** [comment]
+1 ich hab das gerade versucht. Zwar hat er mein Backup importiert, allerdings scheinen manche Dinge die z. B. Die Anzeige der Fehler nicht zu funktionieren. Die Anzahl wird zwar angezeigt aber ich kann sie nicht bestätigen...
